### ISSUE TITLE
Add binary cross entropy loss function

### ISSUE THREAD

**Issue by jonni2** [sbaldu/Neural_network_tracking#4]
I think it would be useful to add the binary cross entropy loss function, which is often used in binary classification problems.
